### PR TITLE
What’s New screen

### PR DESCRIPTION
**Pull request by nummi**
Resolves https://github.com/emberjs/ember-inspector/issues/821

I added a new route and made it the default, `launch`. This route pulls the last loaded version number of Inspector out of local storage and compares it to the current version of Inspector. If the current version is more recent, jump to the What's New screen.

- [x] Load and parse CHANGELOG
- [x] Find _first load_ solution for Chrome (local storage only supported in FF)

Thank for @leongersing for mad regex.

screen shot 2018-08-10 at 8 33 25 pm

### CODE REVIEW

File: app/components/whats-new-content.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+export default Component.extend({
+  error: false,
+  safeHTML: '',
+
+  didInsertElement() {
+    this.element.addEventListener('click', this.handleClick);
+    this.fetchMarkdown.perform();
+  },
+
+  handleClick(e) {
```
Comment (pbishop16) [inline app/components/whats-new-content.js]: Would it be viable to use the native component `click` event handler here?

```
didInsertElement() {
  this.fetchMarkdown.perform();
},

click(e) {
  if (e.target.tagName.toLowerCase() === 'a') {
    e.preventDefault();
    window.open(e.target.href);
  }
},
```

**Comment by nummi** [comment]
> @nummi could we use the chrome storage api to store the info we need?

That seems like a good solution. Should I write a wrapper around local?
https://github.com/emberjs/ember-inspector/blob/master/app/services/storage/local.js

**Comment by nummi** [comment]
I'm starting to wonder if Chrome support is going to be more work than it's worth. The Chrome extension storage service is async so I can't write a simple wrapper that can match the synchronous style of the localstorage and memory storage services.

in `launch.js` I was using the `redirect` method of the Route to decide where to go on app launch:

``` javascript
Route.extend({
  redirect() {
    if(newVersion > storage.get('last-version')) {
      transitionTo('whats-new')
    }
  }
})
```

Would it be too hacky to take advantage of the async handling of the `model` method?

``` javascript
Route.extend({
  model() {
    return new Promise((resolve) => {
      if(Chrome) {
        chromeStorage.get('last-version').then(() => {
          // do check
          resolve();
        })
      } else {
        // localstorage check
        resolve();
      }
    });
  }
})
```

**Comment by rwwagner90** [comment]
@nummi I think perhaps `beforeModel` would be a good place to do these checks and do the appropriate `transitionTo` calls.

**Comment by nummi** [comment]
Okay. Changes made.

Fun fact I learned _after_ finishing the work: Firefox (45 and later) and Edge (14 and later) support  the chrome.storage API. Also the Firefox guides recommend against using localstorage for extensions.

**Comment by rwwagner90** [comment]
This looks mostly good to me. Any chance we could get some basic test coverage in place for this?

**Pull request merged**